spring-boot-autoconfigure/src/main/java/org/axonframework/springboot/TracingProperties.java

```diff
@@ -33,6 +35,17 @@ public class TracingProperties {
      */
     private boolean showEventSourcingHandlers = false;
 
+    /**
+     * Whether to nest spans of subsequent in the same trace. This setting is disabled by default.
```

Comment:
It's valuable to add the defaults here too. Those are then shared in your IDE within `*.properties` files (as part of the documentation, of course).

Reply:
What do you mean? The default is in the javadoc and assigned as a variable, if I'm seeing it correctly

Reply:
If I recall correctly, the IDE grasps the JavaDoc from the fields, not the getters or setters, for `.properties` files.
I am basing that on the `AxonServerConfiguration` within the framework, which only has documentation on the fields.

Reply:
I tried it out, just assigning a default value to the variable makes it show up in the IDE. Isn't this what you mean?

![image](https://user-images.githubusercontent.com/3278878/204077283-c42a82e4-c34d-4a31-99f8-5f31a9f0b899.png)

Reply:
Also works for the duration out of the box
![image](https://user-images.githubusercontent.com/3278878/204077318-38de8097-26be-492f-b9c6-32c0e626fda4.png)

Reply:
However, you are correct in the fact that the javadoc shown comes from the fields, not the getters. Interestingly enough, ctrl-clicking on them takes you to the setter

Reply:
> However, you are correct in the fact that the javadoc shown comes from the fields, not the getters. Interestingly enough, ctrl-clicking on them takes you to the setter

That is most definitely a curious detail. Thanks for sharing that.